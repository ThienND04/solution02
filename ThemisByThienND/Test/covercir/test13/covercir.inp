5
109832027756665539 6691688701424960  9910275210179657
31907815572436602 16697468164952162  16282211281845042
17299572480399040 14943264093085660  9183127775013014
43637906506551483 21016842565160240  18641836303363279
7873808914778389 2019049376690925  629538043175011
