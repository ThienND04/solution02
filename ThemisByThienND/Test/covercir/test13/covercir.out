56525307132710117
16161536227771860
9613002880908477
23006456384174221
4631660124147150
